compiler/noirc_frontend/src/hir/comptime/tests.rs

```diff
@@ -160,6 +160,19 @@ fn for_loop() {
     assert_eq!(result, Value::U8(15));
 }
 
+#[test]
+fn for_loop_inclusive() {
```

Comment:
Could you also add a test in test_programs/execution_success that proves that the new loop bounds work? It can be like this program but with an `assert_eq(...)` at the end. Mainly because the interpreter and the actual execution have different code flows (I'm sure the test will pass, but just to prevent regressions).

Reply:
This is great, thanks for pointing me at these tests 👀 

I added a test to `brillig_loop` in 6eee91b3f8a39cbce19e4a880ba83dd7998b3476
Couldn't find any other test about looping specifically that isn't using the `unconstrainted` keyword, which I assume has something to do with Brillig. 
Do you think this is enough, or should I create a separate test case?

Reply:
Added another to be sure.

Reply:
I think putting it in a separate test is good. Modifying the existing brilling one might be fine too but we'd have to check with the rest of the team: we have some code that checks the number of opcodes produced by program, and by modifying that one it increased (as expected) but I don't know if those programs should not be modified. You could ask Tom or Maxim about this.

Reply:
I would assume that the reports are there to show any unintended increase due to changes in the compiler. I just wasn't sure whether Brillig does things differently in a way that leaves the normal constrained code paths untested. Having both can't hurt :)